app/core/migrations/0027_widget_featured.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 5.0.1 on 2025-09-15 17:33
+
+from django.db import migrations, models
+
+
+def add_featured_flag(apps, schema_editor):
+    Widget = apps.get_model("core", "Widget")
+    db_alias = schema_editor.connection.alias
+    for widget in Widget.objects.using(db_alias).all():
```

Comment:
Only issue I'm concerned of here - upgrades from PHP Materia _may_ contain the `featured` flag prior to the django conversion. It will be part of the FuelPHP database schema in 10.3.2, but not all institutions may be on 10.3.2 prior to moving over. We'll probably want to define a minimum version of the application that can be converted to django, but that's all a bit out of the scope of this PR. In this case, I think we need to:

1. Define the `featured` field in the `001_initial` migration (which represents the FuelPHP DB schema)
2. Define the migration that converts the `featured` PHP field to the desired django version
3. Implement a fallback if the `featured` field is _not_ present (the PHP schema is from a version earlier than 10.3.2), which is more or less what you're doing here.